madmom/utils/__init__.py

```diff
@@ -405,6 +405,37 @@ def quantize_events(events, fps, length=None, shift=None):
     return quantized
 
 
+def load_segments(filename):
+    """
+    Loads labelled segments from a file. Segments are assumed follow the
+    following format, one chord label per line:
+
+    <start_time> <end_time> <chord_label>
+
+    All times should be given in seconds.
+
+    Parameters
+    ----------
+    filename : str or file handle
+        File containing the segments
+
+    Returns
+    -------
+    numpy structured array
+        Structured array with columns 'start', 'end', and 'label', containing
+        the start time, end time, and segment label respectively
+
+    Notes
+    -----
+    Segment files cannot contain comments, because e.g. chord annotations
+    can contain the '#' character! The maximum label length is 64 characters.
+
+    """
+    return np.loadtxt(filename, comments='', dtype=[('start', np.float),
+                                                    ('end', np.float),
+                                                    ('label', 'S64')])
+
+
```

Comment:
Maybe this function should be moved to `features.chords` since it is quite specific to chords.